game/server_main.lua

```diff
@@ -13,7 +13,11 @@ local function spawnPlayer(player)
     local ws = game.world.WORLD_SIZE
     local x = (ws * cs) / 2
     local z = x
-    player:setPosition(x, 100, z);
+    local y = 0
```

Comment:
I'd start from the top and stop when a non-air-block is hit. Otherwise you might spawn the player in a cave (when those are added).

Reply:
I do agree with this, but  as worlds can be infinitely tall it is hard t

Reply:
I see. Maybe use the max height of the terrain generator or something as a starting point?